app/helpers/admin/claims_helper.rb

```diff
@@ -10,13 +10,17 @@ def admin_eligibility_answers(claim)
       claim.policy::EligibilityAdminAnswersPresenter.new(claim.eligibility).answers
     end
 
+    def pii_removed_text
+      content_tag(:span, "Removed", class: "text__quiet")
+    end
+
     def admin_personal_details(claim)
```

Comment:
We have no test coverage for this new behaviour. Worth having some to give us confidence this helper doesn't fall over for a scrubbed claim?